API-Samples/CMakeLists.txt

```diff
@@ -1,5 +1,11 @@
 cmake_minimum_required(VERSION 2.8.11)
 
+# If ANDROID is ON, turn on cross-compiling for it
+if(ANDROID)
+    set(CMAKE_SYSTEM_NAME "Android")
+    set(CMAKE_SYSTEM_VERSION "7")
+endif()
+
```

Comment:
This should work for now, will need to be migrated up to root CMakeLists.txt if/when other parts of this repo add Android support.